apps/mull-ui/src/utilities.ts

```diff
@@ -52,6 +52,19 @@ export const avatarUrl = (user: Partial<User>) =>
     ? `${environment.backendUrl}/api/media/${user.avatar.id}`
     : `./assets/icons/icon-192x192.png`;
 
+export const validateFileSize = (file: File) => {
+  if (!file) {
+    return true;
+  }
+
+  const maxFileSize = 10000000;
```

Comment:
Could you have a variable for this size? similar to the limits I created for the form fields for things such as the event title and description

Reply:
Done